spec/fixtures/files/offline_vm.yml

```diff
@@ -0,0 +1,29 @@
+:apiVersion: kubevirt.io/v1alpha1
+:kind: OfflineVirtualMachine
+:metadata:
+  :name: test
+  :selector:
+    :matchLabels:
+      :flavor: large
+  :namespace: default
+:spec:
+  :template:
```

Comment:
Here you would have a metadat data section with label (as described above)

Reply:
ok